src/cryptography/x509/oid.py

```diff
@@ -4,7 +4,7 @@
 
 import typing
 
-from cryptography.hazmat._oid import ObjectIdentifier
+from cryptography.hazmat._oid import ObjectIdentifier as ObjectIdentifier
```

Comment:
See https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-reexport - seems super-redundant here of course.

Reply:
I kind of hate this, can we not include it in this PR

Reply:
I think we can all agree that this looks very, very, very ugly.

For reference, here's the warning:

```
$ mypy --strict src/cryptography/x509/extensions.py 
src/cryptography/x509/extensions.py:28: error: Module 'cryptography.x509.oid' does not explicitly export attribute 'ObjectIdentifier'; implicit reexport disabled  [attr-defined]
```

I removed this in this PR, but one way or another this has to be solved - see the link I posted above.